src/_newsletters/2017-05-17-newsletter.md

```diff
@@ -7,13 +7,20 @@ date: 2017-05-17 09:00:00 +00:00
 author: Mashooq Badar
 image:
   src: /assets/custom/img/newsletter/newsletter-default.png
+  attribution: 
+    text: Photo by Amador Loureiro on Unsplash
+    href: https://unsplash.com/photos/BVyNlchWqzs
 tags:
 - software craftsmanship
 - TDD
 - testing 
 - conference
 ---
 
+  attribution: 
+    text: Photo by Amador Loureiro on Unsplash
+    href: https://unsplash.com/photos/BVyNlchWqzs
+
```

Comment:
This attribution seems out of place (outside image section)

Reply:
Ooh, good spot. Thanks. Not sure how I did that

Reply:
Resolved in 1b567add